docs/readme.md

```diff
@@ -1,6 +1,7 @@
 Some `*.md` files here are generated from their `*.template.md` counterpart. Generate them with:
 
 ```bash
+$ cd ../
```

Comment:
(FYI scripts work outside the root. But ok it's less confusing to instruct the user to go to the root. Let's keep `cd ../`)

Reply:
Just tried with both npm and yarn and both are fine with me staying in `docs/`. Anyways we can keep the `cd ../` instruction